packages/create-react-native-library/src/index.ts

```diff
@@ -80,7 +89,13 @@ type Answers = {
     | 'kotlin-objc'
     | 'kotlin-swift'
     | 'cpp';
-  type?: 'module-legacy' | 'module-turbo' | 'module-mixed' | 'view' | 'library';
+  type?:
+    | 'module-legacy'
+    | 'module-new-arch'
```

Comment:
@satya164 , I changed 'xxx-turbo' => 'xxx-new-arch'  to avoid any confusion. 
Fabric will refer to view, turbomodule will refer to module. Since the fact fabric is a new renderer, I would like to use `new-arch` postfix for the new arch